ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,224 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  ashenEyeDirections?: string[];
+}
+
+const triggerSet: TriggerSet<Data> = {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
No idea which ZoneId import you mean because line 5 has the ZoneID import I see in other files.

Reply:
Ironically it lets me commit without it just VS Code is stuck with that linter warning 🤔

Reply:
Yeah yeah, I meant that ZoneId import, exactly.  Huh! I don't know why VSCode is confused.  Sometimes I can get it to sort itself out if I close the file and reopen it?  At worst sometimes VSCode needs to be fully closed (if definitions change too much), but I wouldn't expect that to be needed here.

Reply:
Wow you are right after closing and reopening the linter message is gone. I wouldn't have thought that it would have such a bug. In that case it's resolved as well.